nupic/data/stats_v2.py

```diff
@@ -265,7 +265,7 @@ def generateStats(filename, maxSamples = None,):
                            'bool':     BoolStatsCollector,
                            }
 
-  filename = findDataset(filename)
+  filename = resource_filename("examples.prediction.data", filename)
```

Comment:
I don't think this is possible. From the [PkgResource](http://peak.telecommunity.com/DevCenter/PkgResources#basic-resource-access) docs:

> In the following methods, the package_or_requirement argument may be either a Python package/module name (e.g. foo.bar) or a Requirement instance.

Indeed, when I tried it, I got this error:

```
moduleOrReq = <module 'nupic.datafiles' from '/Users/mtaylor/nta/nupic/nupic/datafiles/__init__.pyc'>

>   ???
E   TypeError: __import__() argument 1 must be string, not module

build/bdist.macosx-10.10-intel/egg/pkg_resources.py:335: TypeError
```